### PR TITLE
Port to 21w05b

### PR DESCRIPTION
**Pull request by Motschen**
Just a simple port to 1.17 snapshots.
I tested it, and it seems to work completely fine.
You should probably merge this in a separate 1.17 branch.

### CODE REVIEW

**Comment by deirn** [comment]
nice, just created bai-1.17 branch, can you target that?

**Comment by Motschen** [comment]
Sure :)

File: src/main/java/mcp/mobius/waila/addons/minecraft/HUDHandlerVanilla.java [hunk 1]
```diff
@@ -73,7 +74,7 @@ public void appendHead(List<Text> tooltip, IDataAccessor accessor, IPluginConfig
             MobSpawnerBlockEntity spawner = (MobSpawnerBlockEntity) accessor.getBlockEntity();
             ((ITaggableList<Identifier, Text>) tooltip).setTag(OBJECT_NAME_TAG, new TranslatableText(accessor.getBlock().getTranslationKey())
                 .append(new LiteralText(" ("))
-                .append(spawner.getLogic().getRenderedEntity().getDisplayName())
+                .append(spawner.getLogic().getRenderedEntity(MinecraftClient.getInstance().world).getDisplayName())
```
Comment (deirn) [inline src/main/java/mcp/mobius/waila/addons/minecraft/HUDHandlerVanilla.java:77]: there's `accessor.getWorld()`, so please use that :P

**Review by deirn — CHANGES_REQUESTED** [review]
Just one thing and I'll merge

**Comment by Motschen** [comment]
done ;)

**Comment by deirn** [comment]
uhm, please ignore the actions lmao

**Pull request merged**